src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Added support for configuring the metric exporter's temporality using the
```

Comment:
Remove this [todo](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#:~:text=NOTE%3A%20OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE%20is%20not%20supported%20yet%20%233756.) as well

Reply:
Good call out. I was thinking we should update that README once we have released a version of the package with this feature instead of doing that now.